lib/solidus_sale_prices/engine.rb

```diff
@@ -4,7 +4,7 @@
 
 module SolidusSalePrices
   class Engine < Rails::Engine
-    include SolidusSupport::EngineExtensions::Decorators
+    include SolidusSupport::EngineExtensions
```

Comment:
Typo in commit message: "depreciated" should be "deprecated".

Reply:
Thank you for pointing that out!